### PR TITLE
vlc-video: Enable subtitle track selection

### PR DESCRIPTION
**Pull request by dwot**
Adds options for selecting subtitle track and disabling subtitles from displaying

### CODE REVIEW

File: plugins/vlc-video/vlc-video-source.c [hunk 1]
```diff
@@ -498,6 +514,8 @@ static void vlcs_update(void *data, obs_data_t *settings)
 	size_t count;
 	int network_caching;
 	int track_index;
+	int subtitle_index;
+	int subtitle_enable;
```
Comment (pkviet) [inline plugins/vlc-video/vlc-video-source.c]: why an int and not a bool ?

File: plugins/vlc-video/vlc-video-source.c [hunk 2]
```diff
@@ -915,6 +942,10 @@ static obs_properties_t *vlcs_properties(void *data)
 	obs_properties_add_int(ppts, S_NETWORK_CACHING, T_NETWORK_CACHING,
 			100, 60000, 10);
 	obs_properties_add_int(ppts, S_TRACK, T_TRACK, 1, 10, 1);
+	obs_properties_add_int(ppts, S_SUBTITLE_ENABLE, T_SUBTITLE_ENABLE,
```
Comment (pkviet) [inline plugins/vlc-video/vlc-video-source.c]: obs_properties_add_bool

File: plugins/vlc-video/vlc-video-source.c [hunk 3]
```diff
@@ -870,7 +870,7 @@ static void vlcs_defaults(obs_data_t *settings)
 			S_BEHAVIOR_STOP_RESTART);
 	obs_data_set_default_int(settings, S_NETWORK_CACHING, 400);
 	obs_data_set_default_int(settings, S_TRACK, 1);
-	obs_data_set_default_int(settings, S_SUBTITLE_ENABLE, 0);
+	obs_data_set_default_int(settings, S_SUBTITLE_ENABLE, false);
```
Comment (pkviet) [inline plugins/vlc-video/vlc-video-source.c]: obs_data_set_default_bool

File: plugins/vlc-video/vlc-video-source.c (hunk 1 above)
Comment (dwot) [reply plugins/vlc-video/vlc-video-source.c]: Thank you for this and your work on the audio track update, obviously helped point me in the right places.

File: plugins/vlc-video/vlc-video-source.c (hunk 3 above)
Comment (dwot) [reply plugins/vlc-video/vlc-video-source.c]: Thank you

**Pull request merged**